cmd/fetch_test.go

```diff
@@ -124,29 +129,35 @@ func TestFetchCommand(t *testing.T) {
 		t.Errorf("error mismatch, expect: %s, got: %s", expectErr, err)
 	}
 
+	RemoteHost := fmt.Sprintf("http://localhost:%d", RemotePort)
+
 	// Assign peer A as a remote for peer B
-	cfgCmdText := fmt.Sprintf("qri config set remotes.a_node http://localhost:%d", RemotePort)
+	cfgCmdText := fmt.Sprintf("qri config set remotes.a_node %s", RemoteHost)
 	b.MustExec(t, cfgCmdText)
 
 	// Have peer B fetch from peer A, output correlates to the log from peer A earlier
 	actual = b.MustExec(t, "qri fetch peer_a/test_movies --remote a_node")
-	expect = `1   peer_a/test_movies
```

Comment:
@b5 @dustmop Note the change here. Just to make sure the new output is what is expected to face the user.
The dsLogStringer doesn't show `number of entries`, `number of errors` and whether a particular dataset is `foreign` and finally it doesn't show the `dataset name` at the start of each version.

Reply:
Normally, we shouldn't make UI changes in a refactoring such as this, but in this case it's the correct course of action. I went back to the commit before https://github.com/qri-io/qri/commit/4ce3eaa3228aa6f159bb27bbc1dbda7c6ab2c906 and checked what the output used to be, and validated that you're properly restoring it to what it was. My commit (https://github.com/qri-io/qri/commit/4ce3eaa3228aa6f159bb27bbc1dbda7c6ab2c906) broke it, but `cmd/fetch_test.go` didn't have any expectations that were looking at the actual command output, so that fact wasn't caught.

FYI, it sort of *does* show if the dataset is `foreign`, just in a different way. It says `Storage: remote`, which means the same thing.